### PR TITLE
SC-4652 - User Interface for Account 

### PR DESCRIPTION
**Pull request by luizzaly**
# Description
We implemented the new Design for the Account in the Nuxt client

Main requirements: 
User can change his email address and confirm change via password.
User can change username and password

## Links to Tickets or other pull requests

Base links to copy
- https://github.com/schul-cloud/schulcloud-server/pull/1423
- https://github.com/schul-cloud/schulcloud-server/pull/1356
- https://github.com/schul-cloud/schulcloud-client/pull/1925
- https://ticketsystem.schul-cloud.org/browse/SC-4652
- https://github.com/schul-cloud/integration-tests/pull/84

## Screenshots of UI changes
Opera Momentaufnahme_2020-06-30_115335_localhost
Opera Momentaufnahme_2020-06-30_115830_localhost
Opera Momentaufnahme_2020-06-30_123206_localhost
Opera Momentaufnahme_2020-06-30_115403_localhost
Opera Momentaufnahme_2020-06-30_115422_localhost

## Approval for review

- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definiton of Done

More and detailed information on the _definition of done_ can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

**Comment by ABialoskorski** [comment]
@schul-cloud-bot update screenshots

File: src/components/molecules/AccountCard.vue [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+						<p class="heading">{{ heading }}</p>
+					</slot>
+					<responsive-icon-button
+						v-if="!readonly"
+						design="text"
+						source="material"
+						icon="create"
+						@click="goTo({ targetPath })"
+					>
+						ändern
```
Comment (dominikbruechner) [inline src/components/molecules/AccountCard.vue:19]: missed i18n key here

File: src/pages/account/email/edit.vue [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+			</template>
+		</form-edit-user-data>
+	</section>
+</template>
+
+<script>
+import FormEditUserData from "@components/organisms/FormEditUserData";
+import ModalBodyInfo from "@components/molecules/ModalBodyInfo";
+import ModalFooterConfirm from "@components/molecules/ModalFooterConfirm";
+// import InfoModalFullWidth from "@components/molecules/InfoModalFullWidth";
```
Comment (dominikbruechner) [inline src/pages/account/email/edit.vue:96]: did you leave it commented out on purpose? rather remove the comment, to adhere to clean code

**Comment by BlackGhostHD** [comment]
@schul-cloud-bot update screenshots

**Pull request merged**